### ISSUE TITLE
Removing the keyboard icon in status bar while keeping the keystroke stats? 

### ISSUE THREAD

**Issue by EthanLin-TWer** [KonsomeJona/OctoMouse#16]
Hi there, 

Thanks for making such an interesting app. I don't know why it's just fun. I like it from the moment I downloaded and started using it. The serif font you choose and the svg-like(or just they) icons look delicate. 

I do wanna know if it's possible to retain the keystroke information while removing the keyboard icon, cause two icons takes too much space, and the `0d 00:13:02` timing information doesn't make sense to me and looks distracting. Can one of the icon be removed with the keystroke information kept in someway and made customizable? Would be great to hear your opinions, concerns, etc. 

Thanks,

**Comment by JimmyLv** [comment]
Hi @linesh-simplicity,

I think you can disable timing information in the menu bar:

![image](https://cloud.githubusercontent.com/assets/4997466/26192076/e161d472-3be2-11e7-84ad-f32b40f5097c.png)

**Comment by EthanLin-TWer** [comment]
Yeah, but that disables the 'keystroke' information as well. I want that to give me positive feedback lol

**Comment by livelite** [comment]
I agree, it would be nice to have the option to see a one big number
displayed for keyboard strokes only.
